RepresentsMe/ViewControllers/AppState.swift

```diff
@@ -141,6 +141,10 @@ class AppState {
     /// The listeners for sandbox address changes
     private static var sandboxAddressListeners:[OfficialsListener] = []
 
+    static var openExternalLinksInSafari:Bool = false
+    static var openCalendarOnEventExport:Bool = false
```

Comment:
Can we make these UserDefaults/store on Firebase as a part of User?

Reply:
They are stored on firebase, they are just cached here